spec/lending.md

```diff
@@ -1,13 +1,36 @@
 # Lending
 
 Lending is essential to a margin based protocol. In order to trade margin a user must be able to get the added funds required to over leverage themselves.
-There will be three users of the lending protocol.
 
-First, a user who wants to lend out their money. This user will come to the entropy protocol, add funds to a lending pool. The reason a user will want to do this is because there are users who will want to borrow money in order to margin trade.
+There are three users of the lending protocol.
 
-Second, a user who wants to borrow money. This user will create an account to send money to, this money will represent the collateral for the loan. The c-ratio will be 150%. This means for a loan of $100 the user must put up $150. A user will want to loan money for countless reasons, one being if they believe the underlying collateral token will raise in price.
+- **Lender:** is a user who lends out their money in order to accumulate wealth from the interest paid by borrowers.
 
-Third, a user who wants to conduct a leveraged trade. This user has funds but would like to get margin in order to increase their trade size. This sort of trade will be conducted through a [margin account](./margin.md). A margin account does need to over collateralize. THe reason they do not need to over collateralize is because when a loan is opened the user can not withdraw funds from the account, only add in order to increase the liquidation to current price difference.
+- **Backed Borrower:** is a user who borrow's money that is backed by collateral that is locked up in an obligation account for the duration of the loan. The c-ratio will be 150%. This means for a loan of $100 the user must put up $150. A user will want to loan money for countless reasons, one being if they believe the underlying collateral token will raise in price.
```

Comment:
I'm still not entirely sure of what the use cases of this are? @marbar3778 can you perhaps give me an example of a strategy someone might use?

Reply:
This is the normal lending borrowing design in the blockchain space. Personally, I haven't found much value in this because literally anyone will give you a loan if its over collateralized. It's still good to offer what is available currently.

Reply:
I would assume these would have different interest rates right since the underlying risk is different?

Reply:
yea each pool calculates its own. It's based on the utilization rate. I dont think other projects use volatility in addition to their calculations. A volatility index would be an interesting feature of lending.

Reply:
We can think about adding it at a later time.

Reply:
The only question I would ask is whether the obligation account can be of a different underlying currency i.e. borrow USDC backed by SOL. This would enable shorting